examples/Introduction to Random Distributions.ipynb

```diff
@@ -147,551 +84,353 @@
-      "text/plain": [
-       "<Figure size 1224x360 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "boolean:\n"
+     "ename": "AttributeError",
+     "evalue": "'RandInt' object has no attribute 'pdf'",
```

Comment:
@guillaume-chevalier I'm stuck here. RandInt has no attribute pdf. it's strange running it outside of the notebook works wonder, but it doesn't work in the notebook. This might be related to my version of python. Moving on away from this issue for now. Will revisit later. If you have an idea on what causes this please tell me.

Reply:
I used the following command to create my python3 kernel : 

`python -m ipykernel install --user --name other-env --display-name "Python (other-env)"`